tooling/codelab_rebuild/lib/src/rebuild_blueprint.dart

```diff
@@ -233,6 +233,15 @@ Future<void> _buildBlueprintStep(Directory cwd, BlueprintStep step) async {
     return;
   }
 
+  final stripLinesContaining = step.stripLinesContaining;
+  if (stripLinesContaining != null) {
+    final target = File(p.join(cwd.path, step.path));
+    var lines = target.readAsLinesSync();
```

Comment:
Not important, but seems like the rest of your file uses `final`:

```suggestion
    final lines = target.readAsLinesSync();
```